pkg/frontend/load.go

```diff
@@ -135,14 +136,41 @@ func newNotifyEvent(t notifyEventType, e error, w *WriteBatchHandler) *notifyEve
 }
 
 type PoolElement struct {
+	id        int
 	bat       *batch.Batch
 	lineArray [][]string
 }
 
+type ThreadInfo struct {
+	threadCnt int32
+	startTime time.Time
+}
+
+func (t *ThreadInfo) SetTime(tmp time.Time) {
+	var config atomic.Value
```

Comment:
misuse atomic.Value here. Actually, startTime  should be defined as the atomic.Value. The tmp will be set into the atomic.Value.